docs/general/builtins/aggregation.rst

```diff
@@ -766,12 +767,12 @@ guidelines to keep in mind in this implementation:
 ---------------
 
 Returns the sum of a set of numeric input values that are not ``NULL``.
-Depending on the argument type a suitable return type is chosen. For
-``interval`` argument types the return type is ``interval``. For ``real`` and
-``double precision`` argument types the return type is equal to the argument
-type. For ``byte``, ``smallint``, ``integer`` and ``bigint`` the return type
-changes to ``bigint``. If the range of ``bigint`` values (-2^64 to 2^64-1) gets
-exceeded an ``ArithmeticException`` will be raised.
+Depending on the argument type a suitable return type is chosen. For ``real``,
+``double precision``, ``numeric`` and ``interval`` argument types, the return
```

Comment:
I know you just rephrased it but: By looking at the `SumAggregation`, there is not dedicated registration for `interval`, it seems that it will use the `long` typed impl which returns a `long/bigint`, right?

Reply:
no, there is an `IntervalSumAggregation` dedicated class, I've tested it.

Reply:
ah, missed this, sry!